.env.example

```diff
@@ -11,6 +11,8 @@ CACHE_DRIVER=redis
 SESSION_DRIVER=file
 QUEUE_DRIVER=sync
 
+GLADIATOR_API_KEY=uxm107oct1977
```

Comment:
Yeah, this key is for local dev etc. We'll be setting a more snazzy complex one for prod.